src/components/ebay-radio/examples/3-grouped-radio/template.marko

```diff
@@ -0,0 +1,30 @@
+<fieldset>
+    <legend>Choose an Option</legend>
+    <span class="field">
+        <ebay-radio
+            class="field__control"
+            aria-label="Grouped radio example"
+            id="group-radio-1"
+            value="1"
+            name="radio-group"/>
+        <label class="field__label field__label--end" for="group-radio-1">Option 1</label>
+    </span>
+    <span class="field">
+            <ebay-radio
+                class="field__control"
+                aria-label="Grouped radio example"
+                id="group-radio-2"
+                value="2"
+                name="radio-group"/>
+        <label class="field__label field__label--end" for="group-radio-2">Option 2</label>
+    </span>
+    <span class="field">
+        <ebay-radio
+            class="field__control"
+            aria-label="Grouped radio example"
+            id="group-radio-3"
+            value="3"
+            name="radio-group"/>
+        <label class="field__label field__label--end" for="group-radio-3">Option 3</label>
+    </span>
+</fieldset>
```

Comment:
I think it would be nice if we could add a simple bit of javascript to log the radio-change event to console. Similar I think to how @sharma46bhawana added some script for the hijax pagination example.

Reply:
added for checkbox as well.

Reply:
Changes look good. But in the event detail noticed that `selected` is an integer and not zero index based. This got me thinking that we should make this event consistent with menu & select which returns `index` and `checked`.

Reply:
discussed offline, since checkbox/radio is just single here, don't have access to `array` of inputs, hence can only emit the `value`

Reply:
`selected` is a little ambiguous as a name to end user (i.e. is it selectedIndex or selectedValue or something else?).  And as this thing is the only thing that can be selected (i.e. it's not in a group) I say we rename it to `value`.

We could also just potentially drop it altogether, as it's available in the originalEvent. But let's maybe just rename it and leave it there for now.

Reply:
This got me to thinking that we could, if we wanted, do some clever stuff and sniff out other radio components with the same name attribute. Then we could return the index.

Reply:
updated the property to `value`.

Reply:
reg your second point about group radio - that would be trying to access elements outside the current component - doesn't seem right. Maybe next iteration we could come up `ebay-radio-group`?